lib/logger.js

```diff
@@ -215,7 +225,7 @@ Logger.prototype._bufferLog = function(message) {
             if (err) {
                 debug('Received an error while flushing...' + err);
             }
-        }), configs.FLUSH_INTERVAL);
+        }), this._flushInterval || configs.FLUSH_INTERVAL);
```

Comment:
Once you set `this._flushInterval` above by adding `else` condition, you won't need to use `||` here.